### PR TITLE
updating docs to have a configuration validation section

### PR DESCRIPTION
**Pull request by vi-patel**
## What does this implement/fix? Explain your changes
Documentation update to have a section for the user to verify their configuration settings.

## Does this close any currently open issues?
No.

## More comments
updates the zones query collection

### CODE REVIEW

File: doc/source/configuration.rst [hunk 1]
```diff
@@ -136,4 +136,43 @@ environment variables:
...
+
+Once your settings have been created, run the following command against
+your ManageIQ server to verify your settings are valid.
+
+.. code-block:: bash
+    :linenos:
+
+    (miq-client) $ miqcli --verbose zones query
+
+If you have correct setting to a remote ManageIQ server, your output should
```
Comment (ryankwilliams) [inline doc/source/configuration.rst]: Just a suggestion on the wording here, what do you think?

Replace with: If your settings are correct to a remote ManageIQ server, <rest of sentence>

File: doc/source/configuration.rst [hunk 2]
```diff
@@ -136,4 +136,43 @@ environment variables:
...
+    WARNING: Config file at /etc/miqcli is undefined.
+    WARNING: Config environment variable is undefined.
+    INFO: --------------------------------------------------
+    INFO:                       Zones                       
+    INFO: --------------------------------------------------
+    INFO:  * ID: 2018000000000001
+    INFO:  * Description: Default Zone
+    INFO: --------------------------------------------------
+
+If your settings are incorrect, you will see the an error (please
```
Comment (ryankwilliams) [inline doc/source/configuration.rst]: Just a suggestion on the wording, what do you think?

Replace with: If your settings are incorrect, you will see an error message such as: <rest of sentence>

**Review by ryankwilliams — COMMENTED** [review]
LGTM

Just two minor suggestions on rewording.

**Review by smatula — APPROVED** [review]
LGTM

**Review by ryankwilliams — APPROVED** [review]
LGTM

**Review by tvieira — APPROVED** [review]
LGTM

**Pull request merged**